src/Database.php

```diff
@@ -374,6 +374,14 @@ public static function rawSearchOptionsToAdd()
         return $tab;
     }
 
+    /**
```

Comment:
I'm not fans of duplicated phpdoc when not necessary.

I guess warning will be solved if you use `get_class($item)` interad of `getType()` (probably same for all other ocurrences)

Reply:
This is the only case where using get_class() is sufficient.